ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -160,9 +160,8 @@ ColumnLayout {
         id: displayNamePopupComponent
         DisplayNamePopup {
             profileStore: root.profileStore
-            onClosed: {
-                destroy()
-            }
+            anchors.centerIn: parent
```

Comment:
Do we rely on the `Global.oprnPopup` to set the parent to the main window? AFAIK centering in the Overlay explicitly is the recommended way without creating main window dependency: https://doc.qt.io/qt-5/qml-qtquick-controls2-popup.html#popup-positioning